manual/query_builder/idempotence/README.md

```diff
@@ -116,6 +116,45 @@ Statement statement =
 assert !statement.isIdempotent();
 ```
 
+The generic `remove` method is however safe since collection removals are idempotent:
+
+```java
+Statement statement =
+    update("foo")
+        .remove("l", literal(Arrays.asList(1, 2, 3)))
+        .whereColumn("k").isEqualTo(bindMarker())
+        .build();
+// UPDATE foo SET l=l-[1,2,3] WHERE k=?
+assert !statement.isIdempotent();
+```
+
+When appending, prepending or removing a single element to/from a collection, it is possible to use 
+the dedicated methods listed below; their idempotence depends on the collection type (list, set or 
+map), the operation (append, prepend or removal) and on the idempotence of the element being 
+added/removed:
```

Comment:
maybe we can expand more on this:
>and on the idempotence of the element being  added/removed

(explanation of element idempotence)?

Reply:
Implemented in 5290e35.